source/adminguide/virtual_machines/importing_unmanaging_vms.rst

```diff
@@ -360,6 +360,10 @@ The API has the following preconditions:
 - The Instance state must be 'Running’ or ‘Stopped’
 - The Instance must be a VMware Instance (as of CloudStack 4.19, it's also possible to unmanage a KVM-based Instances)
 
+.. warning::
+   Starting with the 4.22.0 release, when a KVM VM is unmanaged, its infrastructure metadata (zone, pod, cluster, and host) in the libvirt VM XML is preserved.
```

Comment:
```suggestion
   Starting with the 4.22.1 release, when a KVM instance is unmanaged, the infrastructure metadata (zone, pod, cluster, and host details) is preserved in the libvirt VM XML.
```

Note: if any change in the retaining behavior, mainly clearing the metadata while unmanaging instance is handled before 4.22.1, this warning message is not needed and has to be reverted. cc @DaanHoogland